frontend/src/components/Entity.vue

```diff
@@ -1,113 +1,115 @@
-          name: "Permissions",
-          component: markRaw(PermissionList),
-          icon: "security",
-          tooltip: "Manage permissions on the entity"
-        },
-        {
-          name: "History",
-          component: markRaw(Changes),
-          icon: "history",
-          tooltip: 'Change history of entity'
-        }
-      ]
-    };
-  },
-  computed: {
-    title() {
-      return this.entity?.name || this.$route.params.entitySlug || '-';
-    },
-    bindArgs() {
-      return [
-        { schema: this.activeSchema, entity: this.entity },
-        { schema: this.activeSchema, entity: this.entity },
-        { objectType: "Entity", objectId: this.entity?.id },
-        { schema: this.activeSchema, entitySlug: this.$route.params.entitySlug },
-      ]
-    },
+const {getSchema, activeSchema} = useSchema();
+const route = useRoute();
+const entity = ref(null);
+const loading = ref(true);
+const tabs = ref([
+  {
+    name: 'Show/Edit',
+    component: markRaw(EntityForm),
+    icon: "mode_edit",
+    tooltip: "Edit/show entity details"
   },
-  methods: {
-    async getEntity() {
-      if (this.$route.params.entitySlug && this.$route.params.schemaSlug) {
-        const params = {
-          schemaSlug: this.$route.params.schemaSlug,
-          entityIdOrSlug: this.$route.params.entitySlug
-        };
-        this.entity = await this.$api.getEntity(params);
-      } else {
-        this.entity = null;
-      }
-    },
-    async onUpdate(entity) {
-      if (entity) {
-        this.entity = entity;
-      }
-    }
+  {
+    name: "Bulk Add (copy Attributes)",
+    component: markRaw(EntityBulkAdd),
+    icon: "add_circle",
+    tooltip: "Copy over entity attributes to new entities"
   },
-  async activated() {
-    await this.getEntity();
+  {
+    name: "Permissions",
+    component: markRaw(PermissionList),
+    icon: "security",
+    tooltip: "Manage permissions on the entity"
   },
-  watch: {
-    entity(newValue) {
-      if (newValue?.name) {
-        document.title = newValue.name;
-      }
+  {
+    name: "History",
+    component: markRaw(Changes),
+    icon: "history",
+    tooltip: 'Change history of entity'
+  }
+]);
+
+const title = computed(() => {
+  return entity.value?.name || route.params.entitySlug || '-';
+});
+const bindArgs = computed(() => {
+  return [
+    {schema: activeSchema.value, entity: entity.value},
+    {schema: activeSchema.value, entity: entity.value},
+    {objectType: "Entity", objectId: entity.value?.id},
+    {schema: activeSchema.value, entitySlug: route.params.entitySlug},
+  ]
+});
+
+async function getEntity() {
+  if (route.params.entitySlug && route.params.schemaSlug) {
+    const params = {
+      schemaSlug: route.params.schemaSlug,
+      entityIdOrSlug: route.params.entitySlug
+    };
+    entity.value = await api.getEntity(params);
+  } else {
+    entity.value = null;
+  }
+}
+
+async function onUpdate(entity) {
+  if (entity) {
+    entity.value = entity;
+  }
+}
```

Comment:
I have not checked whether this function behaves as intended, but from the looks I would say no:

It looks like this function is setting the `value` attribute of the object in the argument to the object in the argument.

Maybe you would like to rename the argument of this function?

Reply:
Nice catch, thanks!